plugins/georeplication/rest.go

```diff
@@ -1033,7 +1061,11 @@ func georepSSHKeyGenerateHandler(w http.ResponseWriter, r *http.Request) {
 
 	sshkeys, err := getSSHPublicKeys(volname)
 	if err != nil {
-		restutils.SendHTTPError(ctx, w, http.StatusInternalServerError, err)
+		if err == errors.ErrPublicSSHKeyNotFound {
```

Comment:
Nope. Not here.

Reply:
getSSHPublicKeys() handles two errors in its definition same as GetVolume so I guess we can include this error as well in this PR.

Reply:
I don't prefer this PR to change add new error variables if it's used only once.